video.c

```diff
@@ -933,6 +932,14 @@ video_update()
 			mouse_x = event.motion.x;
 			mouse_y = event.motion.y;
 		}
+#if __APPLE__ && (TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE)
+		if (event.type == SDL_DROPFILE) {	   // In case if dropped file
```

Comment:
Can this drop event also be enabled on other systems?
I mean, it would also be nice to have this on my PC.

Reply:
Yes - sure I'll remove the conditional.